ckb/src/main/java/org/nervos/ckb/transaction/CkbTransactionBuilder.java

```diff
@@ -18,6 +18,11 @@ public CkbTransactionBuilder(TransactionBuilderConfiguration configuration, Iter
     super(configuration, availableInputs);
   }
 
+  /**
+   * Add a potential input for the transaction.
+   * <p>
+   * The input may not be actually used if there's already enough capacity for the outputs.
+   */
```

Comment:
OT but this is really surprising. Do we want to change this so that these inputs are definitely used?